gamestonk_terminal/helper_funcs.py

```diff
@@ -608,3 +608,28 @@ def find_tickers(submission):
             l_tickers_found.append(s_ticker.strip())
 
     return l_tickers_found
+
+
+def export_data(export_type, dir_path, func_name, df):
+
```

Comment:
I FINALLY GOT YOU.  WHERE ARE THE DOCSTRINGS

Reply:
> I FINALLY GOT YOU. WHERE ARE THE DOCSTRINGS

To be fair, this was a concept PR :D